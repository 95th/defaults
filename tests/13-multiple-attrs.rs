use defaults::Defaults;

#[derive(Defaults)]
struct Foo {
    #[def = "10"]
    #[def = "100"]
    x: usize,
}

fn main() {}
