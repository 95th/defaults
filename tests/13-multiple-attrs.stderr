error: multiple definitions of `def` found
 --> $DIR/13-multiple-attrs.rs:6:5
  |
6 |     #[def = "100"]
  |     ^^^^^^^^^^^^^^
